### DISCUSSION TITLE
editing g-code for ctb file on elegoo saturn s

### DISCUSSION THREAD

**Discussion by zvsgv** [sn4k3/UVtools#746] [Scripts]
im trying to lift the z axis after specific layers to a higher height and then resume printing in the same layer height. i tried to use dynamic lift height but it seems my printer dosent support this function. 
the end goal is to print with two type of resins in the same layer.
thank you in advance for any respons

**Comment by sn4k3** [comment]
The tool you want to use is the "Timelapse", it allow you to raise Z in specific layer.
Try it first with a dry print on layer 2 for example, to confirm if it works on your printer.

Set "raise each" to 0mm for your purpose
